etc/adminhtml/system.xml

```diff
@@ -63,30 +52,32 @@
                 <field id="enable_intercept" type="select" translate="label comment" sortOrder="1" showInDefault="1" showInWebsite="1" showInStore="1">
                     <label>Send products to Sailthru on product update</label>
                     <source_model>Sailthru\MageSail\Model\Config\Source\ValidatedEnableDisable</source_model>
-                    <comment>This allows us to keep your content library in-sync with Magento.</comment>
+                    <comment><![CDATA[This allows us to keep your content library in-sync with Magento.]]></comment>
                 </field>
                 <field id="send_master" type="select" translate="label comment" sortOrder="2" showInDefault="1" showInWebsite="1" showInStore="1">
                     <label>Send Configurable/Master Products to Sailthru</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
-                    <comment>Should Configurable products (masters) be sent to Sailthru</comment>
+                    <comment><![CDATA[Should Configurable products (masters) be sent to Sailthru]]></comment>
                     <depends>
                         <field id="*/*/enable_intercept">1</field>
                     </depends>
                 </field>
                 <field id="send_variants" type="select" translate="label comment" sortOrder="3" showInDefault="1" showInWebsite="1" showInStore="1">
-                    <label>Send Variant Products to Sailthru</label>
+                    <label>Send Variant Products to Sailthru on Product Update</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
-                    <comment>
-                        Should variant products be sent to Sailthru.
-                        NOTE: If variants aren't visible individually, must enable URL-fragmenting in Sailthru.
-                    </comment>
+                    <comment><![CDATA[Should variant products be sent to Sailthru. NOTE: If variants aren't visible individually, must enable URL-fragmenting in Sailthru.]]></comment>
                     <depends>
                         <field id="*/*/enable_intercept">1</field>
                     </depends>
                 </field>
+                <field id="enable_intercept_cron" type="select" translate="label comment" sortOrder="3" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable Scheduled Products Sync to Sailthru</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                    <comment><![CDATA[Sync Products also Nightly to Sailthru]]></comment>
```

Comment:
Wording

Reply:
@silverman63 let me know what to put here